addons/disposable/XEH_postInit.sqf

```diff
@@ -9,6 +9,11 @@ if (!hasInterface) exitWith {};
     [_unit] call FUNC(updateInventoryDisplay);
 }] call CBA_fnc_addPlayerEventHandler;
 
+["weapon", {
+    params ["_unit","_weapon"];
+    [_unit,_weapon] call FUNC(handlePreparation);
```

Comment:
[`LINKFUNC`](https://github.com/acemod/ACE3/blob/master/addons/main/script_debug.hpp#L7-L15) is used so you can recompile those functions on the fly when developing.